app-modules/ai/src/Filament/Pages/Assistant/Concerns/CanManageThreads.php

```diff
@@ -188,6 +189,10 @@ public function loadFirstThread(): void
         $this->selectThread($this->threadsWithoutAFolder->whereNull('assistant.archived_at')->first());
 
         if ($this->thread) {
+            if ($this->thread->assistant->model->getService() instanceof AiServiceLifecycleHooks) {
+                $this->thread->assistant->model->getService()->afterLoadFirstThread($this->thread);
```

Comment:
Maybe save the thread service into a var so we don't need to call `getService()` twice? Also below